platform/security/core/security-core-services/src/main/java/ddf/security/http/impl/HttpSessionFactory.java

```diff
@@ -36,6 +38,7 @@
   public synchronized HttpSession getOrCreateSession(HttpServletRequest httpRequest) {
     HttpSession session = httpRequest.getSession(true);
     if (session.getAttribute(SecurityConstants.SECURITY_TOKEN_KEY) == null) {
+      session.setMaxInactiveInterval(expirationTime * 60);
```

Comment:
✏️ While it's more verbose (and we didn't do it in the code where it used to live), it might be more clear/correct to do this is `session.setMaxInactiveInterval(TimeUnit.MINUTES.toSeconds(expirationTime));`